.github/workflows/ci.yml

```
@@ -132,6 +76,9 @@ jobs:
 
       - run: echo ${{ steps.ipfs.outputs.cid }}
       - run: echo ${{ github.ref }}
+      - name: Create ipfs-webui.car file
+        run: |
+          ipfs dag export ${{ steps.ipfs.outputs.cid }} > ${{ github.workspace }}/.cache/ipfs-webui@${{ github.sha }}.car
```

nit: Is it still true that `@` is forbidden in filenames on Windows? `_` may be safer here:

```suggestion
          ipfs dag export ${{ steps.ipfs.outputs.cid }} > ${{ github.workspace }}/.cache/ipfs-webui_${{ github.sha }}.car
```